azure_monitor/src/azure_monitor/options.py

```diff
@@ -103,6 +108,10 @@ def _initialize(self) -> None:
         )
         self.endpoint = endpoint + "/v2/track"
 
+        # proxies
+        if self.proxies is None:
+            self.proxies = "{}"
```

Comment:
is not supposed to be typing.Dict[str, str]?

Reply:
Fixed